src/lib/utils.ts

```diff
@@ -1,274 +1,222 @@
+    })),
+  )
+}
 
 export const getLogoUri = async (l1TokenAddress: string) => {
-  const l1TokenAddressLCase = l1TokenAddress.toLowerCase();
-  const logoUri: string | undefined = logoUris[l1TokenAddressLCase];
-  const coinGeckoURI: string | undefined = coingeckoURIs[l1TokenAddressLCase];
-  const trustWalletUri = `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${l1TokenAddress}/logo.png`;
+  const l1TokenAddressLCase = l1TokenAddress.toLowerCase()
+  const logoUri: string | undefined = logoUris[l1TokenAddressLCase]
+  const coinGeckoURI: string | undefined = coingeckoURIs[l1TokenAddressLCase]
+  const trustWalletUri = `https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/${l1TokenAddress}/logo.png`
   const uris = [logoUri, coinGeckoURI, trustWalletUri].filter(
-    (x): x is string => !!x
-  );
+    (x): x is string => !!x,
+  )
 
   for (const uri of uris) {
     try {
-      const res = await axios.get(uri);
+      const res = await axios.get(uri)
       if (res.status === 200) {
-        return uri;
+        return uri
       }
     } catch (e) {}
   }
-  return;
-};
+  return
+}
 export const getTokenListObjFromUrl = async (url: string) => {
-  return (await axios.get(url)).data as TokenList;
-};
+  return (await axios.get(url)).data as TokenList
+}
 export const getTokenListObjFromLocalPath = async (path: string) => {
-  return JSON.parse(readFileSync(path).toString()) as TokenList;
-};
+  return JSON.parse(readFileSync(path).toString()) as TokenList
+}
 
 export const validateTokenList = (tokenList: ArbTokenList | TokenList) => {
-  const ajv = new Ajv();
-  addFormats(ajv);
-  const validate = ajv.compile(schema);
+  const ajv = new Ajv()
+  addFormats(ajv)
+  const validate = ajv.compile(schema)
+
+  const res = validate(tokenList)
+  if (validate.errors) {
+    const output = betterAjvErrors(schema, tokenList, validate.errors, {
+      indent: 2,
+    })
+    console.log(output)
+  }
 
-  return validate(tokenList);
-};
+  return res
+}
 
-export const validateTokenListWithErrorThrowing = (tokenList: ArbTokenList | TokenList)=>{
-  let valid = validateTokenList(tokenList);
-  if (valid) {
-    return true;
-  } else {
-    console.log("Invalid token list:");
-    while (!valid && tokenList.tokens.length > 0){
-      const targetToken = tokenList.tokens.pop()
-      valid = validateTokenList(tokenList);
-      if(valid){
-        console.log('Bad token:', targetToken);
-        throw new Error('Invalid token list due to that token')
-        
-      }
-    }
-    throw new Error('Data does not confirm to token list schema; not sure why');
+export const validateTokenListWithErrorThrowing = (
+  tokenList: ArbTokenList | TokenList,
+) => {
+  let valid = validateTokenList(tokenList)
+  if (valid) return true
+  else {
+    console.log('Invalid token list:')
+    throw new Error('Data does not confirm to token list schema; not sure why')
```

Comment:
typo cofirm/conform (it was mine, but I'm just noticing here lol)

Reply:
Fixed